tests/kas.spec.ts

```diff
@@ -93,8 +89,8 @@ test('test Kafka list filtered by status', async ({ page }) => {
 
   // TODO: Probably race condition when deleting is finished too quickly
   // Currently it just make the test flaky so we should discuss what we will do with it
-  await filterByStatus(page, 'Deleting');
-  await expect(page.getByText(testInstanceName)).toBeTruthy();
+  // await filterByStatus(page, 'Deleting');
+  // await expect(page.getByText(testInstanceName)).toBeTruthy();
```

Comment:
Makes sense, it seems that the instance is deleted quicker than the filter is applied.

Reply:
Correct, I already mentioned that within the previous PR. I will probably revert this before merging and open another PR which will remove it as it is too flaky.